kolena/_utils/consts.py

```diff
@@ -21,3 +21,9 @@ class BatchSize(int, Enum):
 
     LOAD_RECORDS = UPLOAD_RECORDS
     LOAD_SAMPLES = 1_000_000
+
+
+class FieldNamesError(str, Enum):
```

Comment:
nit: This `Enum` isn't an `Error` -- why not just `FieldNames`?

Reply:
Updated to FieldNames